CMakeLists.txt

```diff
@@ -38,21 +38,31 @@ if (PREVENT_DEBUG)
     add_definitions(-DAMB_USE_CLIPBOARD)
 endif()
 
-set(BOOST_ROOT C:/moje/dev/include/boost/boost_1_63_0)
+# BOOST
+set(BOOST_ROOT $ENV{BOOST_ROOT})
+if(NOT BOOST_ROOT)
+  message(FATAL_ERROR "Make sure that BOOST_ROOT environment variable is set.")
+endif()
+
 find_package(Boost REQUIRED)
 if (NOT Boost_FOUND)
     message (ERROR "Boost not found!")
 endif()
 include_directories (${Boost_INCLUDE_DIRS})
 
-set( CMAKE_PREFIX_PATH C:/Qt/Qt5.8.0_x86/5.8/msvc2015/)
+# QT
+set(QT $ENV{QT})
```

Comment:
But this PATHs are not AMB specific. If someone has few different projects and some of them uses paths to QT and / or BOOST in CMake should they really have duplicated environment variables (different names for the same paths)? Because using `AMB_QT` in CMakeLists.txt in different project is not a good idea (in my opinion).